jina/executors/crafters/image/flip.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict
+
+import numpy as np
+
+from .. import BaseCrafter
+from .helper import _restore_channel_axis, _load_image
+
+
+class ImageFlipper(BaseCrafter):
+    """
+    :class:`ImageFlipper` is an abstract class for horizonal and vertical image flip.
+
+    The apply function is :func:`_flip_image`, where the image will be flipped.
+    """
+
+    def __init__(self, channel_axis: int = -1, *args, **kwargs):
+        """
+
+        :param channel_axis: the axis id of the color channel, ``-1`` indicates the color channel info at the last axis
+        """
+        super().__init__(*args, **kwargs)
+        self.channel_axis = channel_axis
+
+    def craft(self, blob: 'np.ndarray', *args, **kwargs) -> Dict:
+        """
+        Flip the input image array.
+
+        :param blob: the ndarray of the image with the color channel at the last axis
+        :return: a chunk dict with the mage
+        """
+        raw_img = _load_image(blob, self.channel_axis)
+        _img = self._flip_image(raw_img)
```

Comment:
```suggestion
        img = np.array(raw_img).astype('float32')
        if self.direction == 'horizon':
            _img = np.fliplr(img)
        elif ....
```

Reply:
Sorry, it was my original suggestion to avoid the if else

Reply:
@hanxiao Instead of direction, can I use a boolean vertical = True/False and flip accordingly? This way, we can avoid any typos?

Reply:
@hanxiao 
Instead of direction, can I use a boolean vertical = True/False and flip accordingly? This way, we can avoid any typos? Like this -
https://github.com/jina-ai/jina/pull/777/commits/64f04bc7eef169f870f0dba26a06fb0adc0e72d6

Reply:
yep

Reply:
Done